lib/docx/body-reader.js

```diff
@@ -300,7 +300,7 @@ function BodyReader(options) {
     
     function readTableRow(element) {
         var properties = element.firstOrEmpty("w:trPr");
-        var isHeader = !!properties.first("w:tblHeader");
+        var isHeader = !!(properties.first("w:tblHeader") || properties.children && properties.children[0].attributes['w:firstRow'] === '1');
```

Comment:
`properties.children[0].attributes` is a list of attributes that is applied to that row of cells. This was the only place in the data that I could find the `w:firstRow` property. It always equals "1" if the "first row" checkbox is checked in the Word table design tools. It always equals "0" if it is not checked.